src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenByIntegratedWindowsAuthParameterBuilder.cs

```diff
@@ -46,6 +46,18 @@ public AcquireTokenByIntegratedWindowsAuthParameterBuilder WithUsername(string u
             return this;
         }
 
+        /// <summary>
+        /// Enables MSAL to read the federation metada for a wstrust exchange from a file instead of acquiring it from an endpoint.
+        /// See aka.ms/MsalFederationMetadata
+        /// </summary>
+        /// <param name="fileName">Path to federation metada xml file.</param>
+        /// <returns></returns>
+        public AcquireTokenByIntegratedWindowsAuthParameterBuilder WithFederationMetadata(string fileName)
```

Comment:
Should be `filePath` here and in other places like `AuthenticationRequestParameters.cs`.

Reply:
Changing to accept metadata directly as a string